bootstrap.ps1

```diff
@@ -79,6 +85,8 @@ Param(
     [switch]$EnableReleaseSymbols,
     [switch]$EnableCoverage,
     [switch]$EnableVerbose,
+    [switch]$EnableAzure,
+    [switch]$EnableGCS,
```

Comment:
Let's remove all references to GCS in the Windows build until we have it supported.

Reply:
Removed.